feat: codex app and cli docs (#2668)
apify/apify-docs: sources/platform/integrations/ai/codex/codex-cli.md

---
title: Codex CLI integration
sidebar_label: Codex CLI
description: Learn how to install the Apify plugin for Codex CLI to discover, run, and build Actors with the Apify MCP server and bundled skills for common workflows.
slug: /integrations/codex-cli
---

import ThirdPartyDisclaimer from '@site/sources/_partials/_third-party-integration.mdx';

[Codex](https://developers.openai.com/codex/) is OpenAI's agentic coding tool. The Codex CLI runs in your terminal, reads and edits your codebase, runs commands, and completes multi-step development tasks.

The [Apify plugin for Codex](https://github.com/apify/apify-codex-plugin) connects Codex to Apify's library of [Actors](https://apify.com/store) and bundles:

> - The [Apify MCP server](/integrations/mcp) for searching the Store, running Actors, and retrieving datasets through the [Model Context Protocol (MCP)](https://modelcontextprotocol.io/docs/getting-started/intro).

Reviewer: Terminology: use `Apify Store` (no article), not "the Store".

```suggestion
- The [Apify MCP server](/integrations/mcp) for searching Apify Store, running Actors, and retrieving datasets through the [Model Context Protocol (MCP)](https://modelcontextprotocol.io/docs/getting-started/intro).
```

- Five built-in skills for common workflows (see [Bundled skills](#bundled-skills) below).

This guide covers installation in the Codex CLI.

<ThirdPartyDisclaimer />

## Prerequisites

- [An Apify account](https://console.apify.com/sign-up) - sign up for free if you don't have one.
- [Codex CLI](https://developers.openai.com/codex/) - installed and authenticated locally.

## Install the plugin

1. In the Codex CLI, run the `/plugins` command.

    ![Codex CLI prompt with the /plugins command typed in](../images/codex-cli/00-plugins-command.webp)

> 1. Use the arrow keys to move right to the **Add Marketplace** tab, then press Enter.

Reviewer: Avoid directional wording ("move right"). "select" reads cleaner and is direction-agnostic.

```suggestion
1. Use the arrow keys to select the **Add Marketplace** tab, then press Enter.
```

    ![Codex CLI plugins menu with the Add Marketplace tab selected](../images/codex-cli/01-add-marketplace-tab.webp)

1. Type the Apify plugin repository and press Enter to confirm:

    ```text
    apify/apify-codex-plugin
    ```

    ![Codex CLI add marketplace prompt with the Apify repository entered](../images/codex-cli/02-marketplace-repo.webp)

1. Open the **Apify Plugin** tab, select **Apify**, and press Enter to view the plugin details.

    ![Codex CLI plugins menu showing the Apify plugin as available](../images/codex-cli/03-plugin-detail.webp)

1. Select **Install plugin** and press Enter to install.

    ![Codex CLI Apify plugin detail with the Install plugin option selected](../images/codex-cli/04-plugin-install.webp)

## Authenticate to Apify

> The plugin bundles the Apify MCP server. Read-only tools like searching the Store and fetching Actor details work without signing in, but you need to authenticate to run Actors and access your account data.

Reviewer: `the Store` → `Apify Store`.

```suggestion
The plugin bundles the Apify MCP server. Read-only tools like searching Apify Store and fetching Actor details work without signing in, but you need to authenticate to run Actors and access your account data.
```
